Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: concourse
-version: 10.0.1
+version: 10.0.4
```

Comment:
This is a breaking change because the variables under `web.service` have been renamed/moved. A users existing `values.yaml` will not be compatible with the new templates. Please make this a major bump to `11.0.0`

Could you also update the `CHANGELOG.md` file stating the reason for the breaking change.